app/scripts/metamask-controller.js

```diff
@@ -3576,6 +3576,7 @@ export default class MetamaskController extends EventEmitter {
       origin: req.origin,
       // This is the default behaviour but specified here for clarity
       requireApproval: true,
+      securityAlertResponse: req.securityAlertResponse,
```

Comment:
Won't have much of an impact, but this could also be code-fenced

Reply:
Done